src/pages/cart/CartNotEmpty.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import CartItem from './CartItem';
+
+const tableHeader = [
+  'Product',
+  'Info',
+  'Price',
+  'Qty.',
+  'Mileage',
+  'Delivery',
+  'Charge',
+  'Total',
+  '선택',
+];
+
+const CartUI = ({ cartList }) => {
+  const price = cartList.map(item => item.price).reduce((a, b) => a + b, 0);
+  const shipping = price >= 50000 ? 0 : 3000;
+  const total = price + shipping;
+
+  const TABLE_FOOTER = [
```

Comment:
```suggestion
  const tableFooter = [
```
이 데이터는 amount가 데이터에 따라서 변하기에 정적인 값이 아닙니다! 정적인 값과 상수데이터 잘 분리해서 생각해주세요!

Reply:
넵🥲